data/json/construction.json

```
@@ -3907,8 +3907,8 @@
     "id": "constr_skylight_frame",
     "description": "Build Sky Light Frame Flat Roof",
```

```suggestion
    "description": "Build Skylight Frame Flat Roof",
```
Skylight is all one word.